contrib/parseq-restli-client/src/test/java/com/linkedin/restli/client/ParSeqRestClientBatchingIntegrationTest.java

```diff
@@ -55,7 +55,7 @@ public void testGetSubResourceRequests() {
     Tuple2Task<Response<Message>,Response<Message>> task = Task.par(associationsGet("a", "b", "x"), associationsGet("a", "b", "y"));
     if (expectBatching()) {
       runAndWaitException(task, RestLiResponseException.class);
-      assertTrue(task.getError().getCause().getMessage().contains("associationsSub?ids=List(x,y)"));
```

Comment:
Why is this change needed? Does it mean that this change is not backwards compatible? For example some users might be calling task.getError().getCause().getMessage().

Reply:
This change is required due to rest.li commit https://github.com/linkedin/rest.li/commit/7498b2196152f64bf36b3f67683673c26228cbdb. The RestLiResponseException.getCause().getMessage will not contain query parameters. This backward incompatibility comes from pegasus (indicated by its major version bump). If the users have bumped to pegasus 19.* or above and their existing code relies on such specific message, they will have to fix this.